### DISCUSSION TITLE
tinyPilot poting to another board

### DISCUSSION THREAD

**Discussion by Avni22** [tiny-pilot/tinypilot#913] [Q&A]
Hi,

I would like to port tinyPilot for another platform expect raspberry pi , for that I have some query related to tinyPilot.

Can anyone help me to resolve the query if possible. It will be heplfull.

Is it possible to port tinypilot for another aarch64 platform? I am planning to port for imx8 board with yocto.

What are the hardware dependencies are there if it is portable?

Is there any meta layer is available for yocto build system ?

is there any docs available for porting ? 

Thanks

**Comment by mtlynch** [comment]
Cross-linking to the forum discussion: https://forum.tinypilotkvm.com/-238/porting-tinypilot-for-another-plateform#post-1
